README.rst

```diff
@@ -353,6 +353,39 @@ the snapshots can be re-enabled for a context manager or in general:
       # snapshotting is disabled only within this block
 
 
+Transaction interceptor
+-----------------------
+
+The ``TransactionInterceptor`` patches Zope's transaction manager in
```

Comment:
@maethu I've done much worse in [production code](https://github.com/4teamwork/opengever.core/blob/0be1b7a24342ff81a95fe1383a4789a37df83ce6/opengever/debug/browser/manage_csrf_tracing.py#L162). 😜